packages/xsnap/src/xsnap.js

```diff
@@ -458,10 +458,10 @@ export async function xsnap(options) {
         const handle = await fs.open(snapPath, 'w+');
         // @ts-expect-error 'close' event added in Node 15.4
         handle.on('close', () => {
-          fs.unlink(snapPath);
+          void fs.unlink(snapPath);
```

Comment:
Maybe add a comment in this case that it's safe to ignore the result because it simply means we may not have cleaned up a temp directory.